docs/pypopminer.md

```diff
@@ -0,0 +1,101 @@
+# pypopminer {#pypopminer}
+
+`pypopminer` is a Python3 package, generated by `boost-python`, which contains MockMiner - data structure, which can mock out VeriBlock + Bitcoin + APM (Altchain POP Miner) completely. 
+
+Use it to generate endorsements of VBK in BTC, and ALT in VBK; generate VBK/BTC blocks with or without endorsements.
+
+### 0. Install dependencies
+
+`pypopminer` depends on [`boost-python`](https://www.boost.org/doc/libs/1_74_0/libs/python/doc/html/building.html), `python3-dev`.
+
```

Comment:
Instructions for specific OS eg Win, Linux, would be very helpful. I'd like to install dependencies with a copy-paste from the doc, not guessing them.